Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM centos:7
+
+RUN yum update -y && \
+  yum install -y centos-release-scl && \
+  yum install -y docker sclo-git212-git unzip java-1.8.0-openjdk-devel java-1.8.0-openjdk-devel.i686 which zip wget make hg svn bzr gcc https://centos7.iuscommunity.org/ius-release.rpm && \
+  yum clean all
+
+ENV PATH=/opt/rh/sclo-git212/root/bin:$PATH
+
+RUN curl --retry 999 --retry-max-time 0  -sSL https://bintray.com/artifact/download/fabric8io/helm-ci/helm-v0.1.0%2B825f5ef-linux-amd64.zip > helm.zip && \
+  unzip helm.zip && \
+  rm -f helm.zip && \
+  mv helm /usr/bin/
+
+# Maven
```

Comment:
Any specific reason why we're using UK mirror here for  maven package?

Reply:
This is inspired from the base image we have currently we used to build it on GKE Jenkins.
https://github.com/fabric8io-images/maven-builder/blob/master/Dockerfile#L37